apps/examples/webserver_test/webserver_test_main.c

```diff
@@ -0,0 +1,1299 @@
+/****************************************************************************
+ *
+ * Copyright 2021 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+/**
+* @brief               To send HTTP request messages and receive response messages.
+*                      supported methods: GET, POST, PUT, DELETE
+*                      Entity and encoding are needed only for POST and PUT methods. If an encoding is not for "C", it always uses the content-length.
+* @scenario            1. Start webserver at TASH using the command "webserver start". Refer to webserver_main.c to run HTTP server.
+*                      2. Start webserver_test at TASH using the command "webserver_test GET http://[serverip]/".
+*
+*****************************************************************************/
+#include <tinyara/config.h>
+#include <tinyara/compiler.h>
+
+#include <sys/types.h>
+#include <sys/socket.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+
+#ifdef CONFIG_NET_LWIP_NETDB
+#include <netdb.h>
```

Comment:
CONFIG_NET_LWIP_NETDB isn't needed to added. because it's checked inside.

Reply:
Made required changes